front/lib/api/chat.ts

```diff
@@ -104,116 +111,102 @@ export async function getChatSession(
   };
 }
 
+export async function getChatMessage(
+  sId: string
```

Comment:
Let's pass the `session: ChatSessionType` here so that we can findAll with the session.id?